From: Robert Richter <robert.richter@amd.com>
Subject: oprofile: remove tracing build dependency
Date: Wed Feb 10 10:03:34 2010 +0100
Git-commit: 309ed95ef0fb36a82d2dfb502d573d7c27504445
Git-repo: git://git.kernel.org/pub/scm/linux/kernel/git/rric/oprofile.git#core
Patch-Mainline: 2.6.34?
Signed-off-by: Tony Jones <tonyj@suse.de>

    [tonyj: note, this also re-enables oprofile on ia64 due to TRACING_SUPPORT
    being previously unsatisfied]

    The commit 1155de4 ring-buffer: Make it generally available
    
    already made ring-buffer available without the TRACING option
    enabled. This patch removes the TRACING dependency from oprofile.
    
    The patch also applies to the 2.6.32-stable kernel.

    Reported-by: Tony Jones <tonyj@suse.de>
    Cc: stable@kernel.org
    Signed-off-by: Robert Richter <robert.richter@amd.com>

diff --git a/arch/Kconfig b/arch/Kconfig
index 9d055b4..25e69f7 100644
--- a/arch/Kconfig
+++ b/arch/Kconfig
@@ -6,8 +6,6 @@ config OPROFILE
 	tristate "OProfile system profiling (EXPERIMENTAL)"
 	depends on PROFILING
 	depends on HAVE_OPROFILE
-	depends on TRACING_SUPPORT
-	select TRACING
 	select RING_BUFFER
 	select RING_BUFFER_ALLOW_SWAP
 	help
